src/pages/components/TaskNew/TaskNew.module.scss

```diff
@@ -0,0 +1,21 @@
+.newItem {
+    display: flex;
+    padding: 15px;
```

Comment:
Input and checkbox not centered. Fix it please

Reply:
fixed styles